### PR TITLE
Fix checkpoint out of bounce?

### PR DESCRIPTION
**Pull request by PlatinMTA**
This should fix the problem with the Checkpoint if i readed the code right. It makes it a lil more tiny for 800x600 resolutions and lower, this should fix the problem but i didn't test it.

### CODE REVIEW

File: resources/[race]/race_sphud/sp_hud.lua [hunk 1]
```diff
@@ -453,7 +453,12 @@ function drawIVHud (player, g_Veh, rank, players, currentCheckpoint, totalCheckp
 			dxDrawRectangle ( i, t, 1, h, tocolor(team[1] or 0, team[2] or 0, team[3] or 0,a) )
 		end
 		
-		dxDrawText ( "CHECKPOINT", l, t, m, b, textColor, labelScale, font, "right", "center", false, false, false, false, true)
+		if sx >= 800 and sy >= 600 then
```
Comment (Bierbuikje) [inline resources/[race]/race_sphud/sp_hud.lua]: '>=' means 'equal to or higher than' so this change would only apply to resolutions lower than 800x600

**Comment by Bierbuikje** [comment]
I'll review this later more in depth, I do not know yet precisely what this does or fixes.

File: resources/[race]/race_sphud/sp_hud.lua (hunk 1 above)
Comment (PlatinMTA) [reply resources/[race]/race_sphud/sp_hud.lua]: Ye, that was the point. First checked highers for doing the regular and using the var, and if it was lower use a custom value that should ajust the 'Checkpoint' getting out of the black background: https://mrgreengaming.com/forums/topic/20104-gui-being-dont-well-displayed/?tab=comments#comment-160235

Comment (Bierbuikje) [reply resources/[race]/race_sphud/sp_hud.lua]: Ah, thanks for clarifying. I'll merge this and thanks for the help!

**Pull request merged**